### PR TITLE
docs(js): Add dist to source maps troubleshooting page

### PR DESCRIPTION
**Pull request by kamilogorek**
Not sure about the wording, but the content itself is valid.

### CODE REVIEW

File: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx [hunk 1]
```diff
@@ -31,6 +31,18 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
...
 
+<PlatformSection notSupported={["javascript.electron"]}>
+
+## Verify artifacts distribution match value configured in your SDK
+
+Whenever you are using distribution identifier (`dist` configuration option in the SDK), it has to be matched during the source maps upload as well.
+To do this, use `--dist` flag in the `sentry-cli` or `dist` option if you are using `@sentry/webpack-plugin`, and make sure that the same value is used in both places.
+
+To verify that the distribution has been used and set correctly, open up the issue from the Sentry UI and check if the `dist` tag is present.
+For the artifacts, go to the `Source Maps` page in the project settings, and verify that required files have this value set. If not, it will say `none` instead.
```
Comment (AbhiPrasad) [inline src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx:42]:
```suggestion
For artifacts, go to the `Source Maps` page in the project settings, and verify that the required files have this value set. If the value is not set, the files will say `none` instead.
```

File: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx [hunk 2]
```diff
@@ -31,6 +31,18 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
+<PlatformSection notSupported={["javascript.electron"]}>
+
+## Verify artifacts distribution match value configured in your SDK
+
+Whenever you are using distribution identifier (`dist` configuration option in the SDK), it has to be matched during the source maps upload as well.
```
Comment (AbhiPrasad) [inline src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx:38]:
```suggestion
Whenever you are using a distribution identifier (`dist` configuration option in the SDK), the same value has to be used during the source maps upload as well.
```

File: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx [hunk 3]
```diff
@@ -31,6 +31,18 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
+<PlatformSection notSupported={["javascript.electron"]}>
+
+## Verify artifacts distribution match value configured in your SDK
```
Comment (lobsterkatie) [inline src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx:36]:
```suggestion
## Verify artifact distribution value matches value configured in your SDK
```

File: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx [hunk 4]
```diff
@@ -31,6 +31,18 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
...
+
+Whenever you are using distribution identifier (`dist` configuration option in the SDK), it has to be matched during the source maps upload as well.
+To do this, use `--dist` flag in the `sentry-cli` or `dist` option if you are using `@sentry/webpack-plugin`, and make sure that the same value is used in both places.
```
[the remark this answers is not in the document]
Comment (lobsterkatie) [reply src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx:39]: We just told them above that the values need to match, so I don't think we need to say it again.

```suggestion
To add a `dist` value to your uploaded source maps, use the `--dist` flag with `sentry-cli` or the `dist` option with `@sentry/webpack-plugin`. To set the `dist` value in the SDK, use the `dist` option in your `Sentry.init()`.
```

File: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx (hunk 1 above)
Comment (lobsterkatie) [inline src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx:42]: The style guide now puts all sentences in a paragraph on the same code line, for better wrapping. Also, I don't think we need to mention "none" if we've told them where to look - it's pretty self-explanatory.

```suggestion
To verify that the distribution has been set correctly in the SDK, open an issue in the Sentry UI and check that the `dist` tag is present. For artifacts, go to the `Source Maps` page in project settings, choose the release shown on the event you just checked, and verify that the `dist` value (in the small oval next to the upload time) matches that on the event.
```

File: src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx (hunk 2 above)
Comment (lobsterkatie) [reply src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx:38]: Building on Abhi's suggestion (so this includes his and mine).

```suggestion
Whenever you are using a distribution identifier (the `dist` configuration option in the SDK), the same value has to be used during source map upload. Conversely, if your source maps are getting uploaded with a `dist` value, the same value must be set in your SDK.
```

**Review by lobsterkatie — COMMENTED** [review]
Thanks for finally doing this, @kamilogorek!

**Pull request merged**